model/src/main/proto/preprocess_protos.bzl

```diff
@@ -0,0 +1,21 @@
+
+def process(name, src):
```

Comment:
Here & elsewhere (including the filename), can we come up with a more descriptive name than 'process'? What specifically is being processed in the src object, and what is outputted? Given that this is wrapping ``proto_library``, I'd expect us to keep 'proto' and 'library' somewhere in the name.

Reply:
How about 'preprocess_path_proto_library' or 'format_import_proto_library'? I definitely need more practice coming up with adequately descriptive names!

Reply:
I changed the name to `format_import_proto_library` but please let me know if you think I should name it something else!

Reply:
This name seems good--thanks!